src/gleam/float.gleam

```diff
@@ -384,3 +384,17 @@ if javascript {
   external fn do_random_uniform() -> Float =
     "../gleam_stdlib.mjs" "random_uniform"
 }
+
+pub type DivisionByZero {
+  DivisionByZero
+}
+
+pub fn divide(
+  dividend dividend: Float,
+  divisor divisor: Float,
```

Comment:
We don't use nouns for labels in the standard library, so let's change this one to "by" and remove the first label.

Reply:
If there is a chance to keep this, the pattern could be:
0..n nouns followed by 0..n adjectives/, example:

```gleam
stepper(min: 0, max: 20, stepping: 5) // [0, 5, 10, 15, 20]
```

Reply:
I've removed the label from the first param.